### ISSUE TITLE
'Push me' button which is present in the table is not keyboard accessible in Windows.

### ISSUE THREAD

**Issue by msftedad** [ceolter/ag-grid#9871]
**Title:** 'Push me' button which is present in the table is not keyboard accessible in Windows.

**Test Environment**
Windows: Windows 11 Version 24H2 (OS build 26120.2705)
Browser Edge Version 133.0.3065.51 (Official build) (64-bit)

**User Impact**
The "Push Me" button within the table is not accessible via the keyboard on Windows, preventing users who rely on keyboard navigation from interacting with it. This impacts accessibility compliance and usability for keyboard-only and assistive technology users.

**Observation**
Push me buttons are accessible using mouse.

**Repro Steps**

1. Open URL: [JavaScript Grid: Quick Start | AG Grid](https://www.ag-grid.com/javascript-data-grid/getting-started//#example) in Edge browser.
2. Press tab key to navigate to the table below Example heading.
3. Press tab key to navigate to 'Push me' button present under 'Button' column.
4. Observe with Enter key whether 'Push me' button present in the table under Button column is keyboard accessible or not.

**Actual Result**
'Push me' button which is present in the table is not keyboard accessible in Windows. But when clicked with mouse the push me button is getting activated and clicked dialog is appearing.

**Refer Attachment:**
'Push me' button which is present in the table is not keyboard accessible in Windows.

https://github.com/user-attachments/assets/98837021-22df-4cf3-a87c-a087f4d8afab

<img width="957" alt="Image" src="https://github.com/user-attachments/assets/bf9384a2-b05e-4efa-8211-ee8ac0f338af" />

**Expected Result**
'Push me' button which is present in the table should be keyboard accessible when pressed Enter/Space key.

--------------------------------------------------------------------------------------------------------------------------------------

**Comment by AG-Zoheil** [comment]
Hi,

Thanks for raising this via GitHub.

Please see this page in our documentation to understand how to achieve accessibility for this: https://www.ag-grid.com/javascript-data-grid/keyboard-navigation/#custom-cell-component.

Kind regards,
Zoheil
